core/commands: fix references to all commands page (#32)
botlabs-gg/yagpdb-docs-v2: content/reference/templates/_index.md

0x22 > `"` for straight double quotes, 0x27 > `'`for apostrophes and 0x60 `` ` `` for backticks/back quotes; so make
sure no "smart-quotes" are being used.

> The difference between back quotes and double quotes in string literals is covered in
> [Golang reference documentation](https://go.dev/ref/spec#String\_literals).

Reviewer: ```suggestion
The difference between back quotes and double quotes in string literals is covered in
the [Go Language Specification](https://go.dev/ref/spec#String\_literals).
```

The name of this popular programming language is Go, the name of the linked document is its language specification.
We have more instances of "Golang" instead of "Go"; please change those as well.

|.IsMessageEdit| Returns boolean true/false if message is edited and edit trigger for custom commands is enabled. Defaults to false.|
|.IsPremium| Returns boolean true/false whether guild is premium of YAGPDB or not.|
|.LinkRegex| Returns string value of in-built link-matching regular expression.|

> |.Permissions| Returns all mapped-out permission bits available for Discord in their bitshifted decimal values; <br>e.g. `{{.Permissions.AddReactions}}` would return `64`, same as `{{bitwiseLeftShift 1 6}}`. More in [Discord Permissions documentation](https://discord.com/developers/docs/topics/permissions#permissions).|

Reviewer: ```suggestion
|.Permissions| Returns all mapped-out permission bits available for Discord in their bitshifted decimal values; <br>e.g. `{{.Permissions.AddReactions}}` would return `64`, same as `{{bitwiseLeftShift 1 6}}`. More in [Discord's Permissions documentation](https://discord.com/developers/docs/topics/permissions#permissions).|
```
Whilst both approaches seem correct, I think this way it is clearer that this is

1. an external link
2. said external link goes to Discord's documentation
3. we have nothing to do with what's on that page

|.Channel.NSFW| Outputs whether this channel is NSFW or not.|
|.Channel.OwnerID| The ID of the creator of threads as _int64_. Returns `0` for normal channels.|
|.Channel.ParentID| The ID of the channel's parent (category), returns 0 if none.|
|.Channel.PermissionOverwrites| A slice of [Discord permission overwrite](https://discord.com/developers/docs/resources/channel#overwrite-object) structures applicable to the channel.|
|.Channel.Position| Channel position from top-down.|
|.Channel.Topic| The topic of the channel.|

> |.Channel.Type| The type of the channel. Explained further in [Discord channel documentation](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)|

Reviewer: ```suggestion
|.Channel.Type| The type of the channel. Explained further in [Discord's channel documentation](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)|
```
Same reasoning as before. There may be / are more instances of this, but I don't want to flood this conversation with noise over an `'s`. Please go over similar links scoped to this PR and try to determine whether it makes more sense to have them in a possessive form, or to keep them as-is.
